### PR TITLE
Fix version numbers for the new 0.9.0-rc1 release [ECR-3963]

### PR DESCRIPTION
**Pull request by vitvakatu**
Are we going to release `0.9.0`, not `0.9.0-rc1` or smth?

### CODE REVIEW

**Comment by dmitry-timofeev** [comment]
0.9.0-rc1 , see the first section of https://wiki.bf.local/display/EJB/Java+Binding+0.9.0-rc1+Release+Checklist

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -970,22 +970,22 @@ An example of API service tests can be found in
...
-[testkit-extension-validator]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/Validator.html
-[testkit-extension-validatorcount]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/ValidatorCount.html
-[testkit-fake-time-provider]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/FakeTimeProvider.html
-[testkit-find-in-pool]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/TestKit.html#findTransactionsInPool(java.util.function.Predicate)
-[testkit-get-pool]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/TestKit.html#getTransactionPool()
-[testkit-get-port]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/TestKit.html#getPort()
-[testkit-maven]: https://mvnrepository.com/artifact/com.exonum.binding/exonum-testkit/0.8.0
-[testkit-time-provider]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/TimeProvider.html
+[node-submit-transaction]: https://exonum.com/doc/api/java-binding/0.9.0-rc1/com/exonum/binding/core/service/Node.html#submitTransaction(com.exonum.binding.transaction.RawTransaction)
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: It seems to be duplicated.

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -1466,46 +1466,46 @@ For using the library just include the dependency in your `pom.xml`:
...
-[transactionconvererter]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/core/service/TransactionConverter.html
+[transactionconvererter]: https://exonum.com/doc/api/java-binding/0.9.0-rc1/com/exonum/binding/core/service/TransactionConverter.html
 [vertx-web-docs]: https://vertx.io/docs/vertx-web/java/#_basic_vert_x_web_concepts
 [maven-install]: https://maven.apache.org/install.html
-[cryptofunctions-ed25519]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/common/crypto/CryptoFunctions.html#ed25519--
-[service-create-public-api]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/core/service/Service.html#createPublicApiHandlers-com.exonum.binding.service.Node-io.vertx.ext.web.Router-
-[transaction-result]: https://docs.rs/exonum/0.12/exonum/blockchain/struct.TransactionResult.html
+[cryptofunctions-ed25519]: https://exonum.com/doc/api/java-binding/0.9.0-rc1/com/exonum/binding/common/crypto/CryptoFunctions.html#ed25519--
+[service-create-public-api]: https://exonum.com/doc/api/java-binding/0.9.0-rc1/com/exonum/binding/core/service/Service.html#createPublicApiHandlers-com.exonum.binding.service.Node-io.vertx.ext.web.Router-
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: https://exonum.com/doc/api/java-binding/0.9.0-rc1/com/exonum/binding/core/service/Service.html#createPublicApiHandlers(com.exonum.binding.core.service.Node,io.vertx.ext.web.Router)

(probably, link format has been changed in some javadoc release)

File: src/get-started/java-binding.md [hunk 3]
```diff
@@ -1466,46 +1466,46 @@ For using the library just include the dependency in your `pom.xml`:
...
+[transaction-result]: https://docs.rs/exonum/0.13.0-rc.2/exonum/blockchain/struct.TransactionResult.html
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: 404

Comment (vitvakatu) [reply src/get-started/java-binding.md]: It is not used anymore

**Pull request merged**